ourrevolution/settings.py

```diff
@@ -279,6 +279,10 @@
 NOMINATIONS_QUESTION_ISSUES_ID = 24
 
 ADD_GROUP_URL = '/groups/new/'
+ELECTORAL_COORDINATOR_EMAIL = os.environ.get(
+    'ELECTORAL_COORDINATOR_EMAIL',
+    'political@ourrevolution.com'
```

Comment:
It's good we are not putting anyone's personal email into the codebase :)

I suggest to make a setting for political@ourrevolution.com if we don't have one already.